### PR TITLE
Cryptokit 1.18

### PR DESCRIPTION
**Pull request by xavierleroy**
Release 1.18:
- Add BLAKE3 hash and MAC functions.
- Fix compile-time error "SSE4.1 instruction set not enabled".

### CODE REVIEW

File: packages/cryptokit/cryptokit.1.18/opam [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    "@doc" {with-doc}
+  ]
+]
+dev-repo: "git+https://github.com/xavierleroy/cryptokit.git"
+url {
+  src: "https://github.com/xavierleroy/cryptokit/archive/release118.tar.gz"
+  checksum: [
+    "sha256=0e9d636dc45967ac3529831ec20aed1aff4c941ae6cfb69b7c29c3d721db2c2b"
+    "sha512=3d797d250941f6b3f991b52238509030aa3a21f0fa4967eb6c9aa1a2d3f1faeac9ddfb088d999b768dce3768741158a5baea386f55a641d09b7a0c148ec87ac9"
+  ]
```
Comment (mseri) [inline packages/cryptokit/cryptokit.1.18/opam:38]:
```suggestion
  ]
}
```

Comment (xavierleroy) [reply packages/cryptokit/cryptokit.1.18/opam]: Sure, sorry for this typo.

**Comment by xavierleroy** [comment]
Am I expected to change something in this PR before it can be accepted?  I had a quick look at the first failures reported by CI, and they seem unrelated to Cryptokit, e.g. a C include or C library wasn't found, that kind of things.

**Comment by mseri** [comment]
Apologies for the massive delay. I wanted to use the revdeps failures to track missing ocaml 5 bounds, but then it remained lagging behind. I went through the failures and they seem to be unrelated

**Comment by xavierleroy** [comment]
Thanks a lot !

**Comment by avsm** [comment]
Thanks @mseri -- I spotted a few more OCaml 5 failures in the revdeps and filed some followup PRs to restrict them.

**Pull request merged**